pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -381,6 +379,7 @@ func handleNetworkInstanceDelete(ctxArg interface{}, key string,
 		doNetworkInstanceInactivate(ctx, status)
 	}
 	doNetworkInstanceDelete(ctx, status)
+	appNumOnUNetBaseDelete(status.UUID)
```

Comment:
When we get to this point do we know that all of the uuidpairtonum for this bridge have already been deleted?
Or can there be AppNetworkStatus which still implicitly depend on the NI?
Perhaps there needs to be a check at the top of handleNetworkInstanceDelete whether any network instance depends on it? Otherwise couldn't strange things happen inside uuidpairtonum if the AppNetworkStatus are deleted after the NI?